android/build.gradle

```diff
@@ -46,4 +46,8 @@ allprojects {
             url "$rootDir/../node_modules/detox/Detox-android"
         }
     }
+
+    ext {
+        set("firebaseMessagingVersion", "21.0.0")
```

Comment:
Maybe add the same TODO: here?

Reply:
No point poluting user product with internal todos imo

Reply:
Aside of even when widget resources will be updated; we will still have to have this one here for backwards compat

Reply:
well rethought it let's add a comment at least